lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -25,6 +26,19 @@ const NetworkRequest = require('../../lib/network-request');
 // to possibly be non-blocking (and they have minimal impact anyway).
 const MINIMUM_WASTED_MS = 50;
 
+const UI_STRINGS = {
```

Comment:
wdyt about `strings`?   the yelling hurts me. :(

Reply:
I'm on board for anything we reasonably think will never be used in any other context, I'd love to add another script for essentially linting to make sure all UI_STRINGS get exported, all get stuck into `i18n` functions, etc.